django/db/backends/base/schema.py

```diff
@@ -121,6 +121,15 @@ def __exit__(self, exc_type, exc_value, traceback):
 
     # Core utility functions
 
+    def prepare_param(self, node):
+        sql, params = None, (node,)
```

Comment:
This assignment and tuple creation /  unpacking, etc. isn't used in the case that `hasattr` is true. I also noted this below, but the conversation is marked resolved.